tests/graceful-recovery/graceful-recovery.md

```diff
@@ -34,7 +34,6 @@ Ensure that NGF can recover gracefully from container failures without any user
 3. Check out the latest tag (unless you are installing the edge version from the main branch).
 4. Go into `deploy/manifests/nginx-gateway.yaml` and change the following:
 
-   - `runAsNonRoot` from `true` to `false`: this allows us to insert our ephemeral container as root which enables us to restart the nginx-gateway container.
```

Comment:
Why this change?

Reply:
Sorry I may have mislead Saloni. You don't need to change anything from the original `graceful-recovery.md`, its just that the original document no longer matches the automated test. Which is okay for now I think.